notification-svc/index.ts

```diff
@@ -1,6 +1,15 @@
 import express, { Express } from 'express'
 import cors from 'cors'
 import reschedulingPromptRoutes from './routes/rescheduling-prompt'
+import confirmationJob from './models/confirmationJob'
+import crypto from 'crypto';
+
+// Generate a 256-bit (32-byte) secret key for AES-256 encryption
+const secretKey = crypto.randomBytes(32);
+// Generate a 128-bit (16-byte) IV for AES
+
+console.log('Secret Key:', secretKey.toString('hex'));
+
```

Comment:
I think you should put secret key in .env and also delete console.log

Reply:
My bad, I did not mean to leave it there

Reply:
secret key will not be changing often and will be kept in notification-svc's and appointment-svc's .env files